### PR TITLE
MoH Niger CHW - Generate the APK

### PR DESCRIPTION
**Pull request by ernestoteo**
[#51](https://github.com/medic/config-moh-niger/issues/51)

### CODE REVIEW

**Comment by binokaryg** [comment]
Have you shared the keys somewhere?

File: src/moh_niger_chw/res/values/strings.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+	<string name="app_name">RCOM</string>
+	<string name="app_host">moh-niger-chw.dev.medicmobile.org</string>
```
Comment (binokaryg) [inline src/moh_niger_chw/res/values/strings.xml:4]: We don't have the production URL yet?

File: src/moh_niger_chw/res/values/strings.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+	<string name="app_name">RCOM</string>
```
Comment (binokaryg) [inline src/moh_niger_chw/res/values/strings.xml:3]: Name of the app according to the GH issue is:
`RCOM (Relais Communautaire)`

According to the docs linked, the app name is:
`RCom ( Relais Communautaire )`

Please confirm that `RCOM` is what we need.

Comment (binokaryg) [reply src/moh_niger_chw/res/values/strings.xml]: cc @hiellavis

Comment (hiellavis) [reply src/moh_niger_chw/res/values/strings.xml]: Hi @binokaryg 
The name of the application is `RCOM (Relais Communautaire)`.
Thx

**Comment by latin-panda** [comment]
To cross post, we're working on the play store issue in this [ PR ](https://github.com/medic/cht-android/pull/283), the ticket is linked for more details. I'll let you know once the issue is fixed :)

**Comment by ernestoteo** [comment]
> To cross post, we're working on the play store issue in this [ PR ](https://github.com/medic/cht-android/pull/283), the ticket is linked for more details. I'll let you know once the issue is fixed :)

Thanks @latin-panda

**Review by adrienlemec — APPROVED** [review]

**Comment by latin-panda** [comment]
Thank you, @ernestoteo please cherry pick this commit and send the PR to master, so this flavor gets maintained by the team :) 
After the cherry pick PR is squash-merged, no more actions is required.

**Pull request merged**